.github/workflows/release.yml

```diff
@@ -118,38 +117,12 @@ jobs:
           echo "Publish docs to Confluence: $PUBLISH_DOCS"
           echo "Append notes to Confluence: $APPEND_NOTES"
           echo "Commit details: $COMMIT_DETAILS"
```

Comment:
- The `Append Fix Notes to Confluence` step processes `new_fix_notes` for formatting but the combination with existing content should be done carefully to avoid any formatting errors.